src/settings.ts

```diff
@@ -63,6 +64,20 @@ export class TrelloSettings extends PluginSettingTab {
       });
   }
 
+  private openInDesktopSetting(containerEl: HTMLElement, settings: PluginSettings): void {
+    if (Platform.isMacOS || Platform.isWin) {
+      this.plugin.log('TrelloSettings.openInDesktopSetting', '-> Adding open in desktop config setting');
+      new Setting(containerEl)
+        .setName('Open in Trello desktop')
+        .setDesc('Opens Trello card in desktop application on macOS or Windows')
```

Comment:
Just some minor copy changes, i feel like this flows a bit better
```suggestion
        .setDesc('Opens cards in Trello desktop application on macOS or Windows')
```